src/decimal-input/index.jsx

```diff
@@ -77,7 +78,7 @@ class DecimalInput extends React.Component {
 				must be an integer or period character`);
 		}
 
-		let value = this.state.value === Number(0).toFixed(this.props.places) ?
+		let value = this.state.value === Numbers.toFixed(Number(0), this.props.places) ?
```

Comment:
I only left this as I wanted to change as little as possible. I'll drop it out now...